### PR TITLE
Add exception for single ridge in Voronoi diagram

### PR DESCRIPTION
**Pull request by mxwell**
Please, review the changes.

### CODE REVIEW

File: centerline/io.py [hunk 1]
```diff
@@ -54,11 +56,18 @@ def create_centerlines(src, dst, density=0.5):
...
-                        interpolation_dist=density,
-                        **attributes
-                    )
+                    try:
+                        centerline_obj = Centerline(
+                            input_geom=input_geom,
+                            interpolation_dist=density,
+                            **attributes
+                        )
+                    except RuntimeError as err:
```
[the remark this answers is not in the document]
Comment (mxwell) [reply centerline/io.py]: I think it got covered in later commits.

File: centerline/main.py [hunk 2]
```diff
@@ -92,6 +92,13 @@ def _create_centerline(self):
                 if line.within(self._input_geom) and len(line.coords[0]) > 1:
                     lst_lines.append(line)
 
+        nr_lines = len(lst_lines)
+        if nr_lines < 2:
+            raise RuntimeError(
+                "Number of produced ridges is too small: {}".format(nr_lines) +
```
[the remark this answers is not in the document]
Comment (mxwell) [reply centerline/main.py]: Fixed

File: tests/test_main.py [hunk 3]
```diff
@@ -83,6 +83,15 @@ def test__geometry_collection__raises_valueerror(self):
...
             Centerline(GEOMETRY_COLLECTION)
 
+    def test__too_large_interp_dist__raises_runtimeerror(self):
+        POLYGON = Polygon([[0, 0], [10, 0], [10, 10], [0, 10]])
+
+        with self.assertRaises(RuntimeError):
+            Centerline(POLYGON, 10)
+
+        centerline = Centerline(POLYGON, 5)
+        self.assertIsInstance(centerline, Centerline)
```
[the remark this answers is not in the document]
Comment (mxwell) [reply tests/test_main.py]: The point is that there is also a smaller `interpolation_dist` value that doesn't raise exceptions.

[the remark this answers is not in the document]
Comment (fitodic) [reply tests/test_main.py]: Isn't that already covered by `test__polygon__returns_multilinestring`?

File: centerline/io.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (fitodic) [reply centerline/io.py]: You're right :+1:

**Comment by fitodic** [comment]
@mxwell Thanks for the contribution! Merged :slightly_smiling_face:

**Pull request merged**